cli/CHANGELOG.md

```diff
@@ -23,16 +23,6 @@ _14 May 2025_
 ### Documented
 
 * Replace old c2patool release notes with CHANGELOG ([#1063](https://github.com/contentauth/c2pa-rs/pull/1063))
-<div class="github-only" >
```

Comment:
Gah, I didn't see _that_ part.

Reply:
I use that in various places in markdown files to have content that we don't want to display in the doc site, but DO want in GitHub.  You don't see the tag itself in either place (only if you edit the md file).  Anyway, I removed it for now.  The issue was the lack of a new line before it, which apparently breaks the build due to linting rules.